bundles/org.openhab.binding.electroluxair/src/main/java/org/openhab/binding/electroluxair/internal/handler/ElectroluxAirHandler.java

```diff
@@ -0,0 +1,226 @@
+
+/**
+ * The {@link ElectroluxAirHandler} is responsible for handling commands, which are
+ * sent to one of the channels.
+ *
+ * @author Jan Gustafsson - Initial contribution
+ */
+@NonNullByDefault
+public class ElectroluxAirHandler extends BaseThingHandler {
+
+    private final Logger logger = LoggerFactory.getLogger(ElectroluxAirHandler.class);
+
+    private ElectroluxAirConfiguration config = new ElectroluxAirConfiguration();
+
+    public ElectroluxAirHandler(Thing thing) {
+        super(thing);
+    }
+
+    @Override
+    public void handleCommand(ChannelUID channelUID, Command command) {
+        logger.debug("Command received: {}", command);
+        if (CHANNEL_STATUS.equals(channelUID.getId()) || command instanceof RefreshType) {
+            update();
+        } else {
+            ElectroluxPureA9DTO dto = getElectroluxPureA9DTO();
+            ElectroluxDeltaAPI api = getElectroluxDeltaAPO();
+            if (api != null && dto != null) {
+                if (CHANNEL_WORK_MODE.equals(channelUID.getId())) {
+                    if (command.toString().equals(COMMAND_WORKMODE_POWEROFF)) {
+                        api.workModePowerOff(dto.getPncId());
+                    } else if (command.toString().equals(COMMAND_WORKMODE_AUTO)) {
+                        api.workModeAuto(dto.getPncId());
+                    } else if (command.toString().equals(COMMAND_WORKMODE_MANUAL)) {
+                        api.workModeManual(dto.getPncId());
+                    }
+                } else if (CHANNEL_FAN_SPEED.equals(channelUID.getId())) {
+                    api.setFanSpeedLevel(dto.getPncId(), Integer.parseInt(command.toString()));
+                } else if (CHANNEL_IONIZER.equals(channelUID.getId())) {
+                    if (command == OnOffType.OFF) {
+                        api.setIonizer(dto.getPncId(), "false");
+                    } else if (command == OnOffType.ON) {
+                        api.setIonizer(dto.getPncId(), "true");
+                    } else {
+                        logger.debug("Unknown command! {}", command);
+                    }
+                }
+            }
+        }
+    }
+
+    @Override
+    public void initialize() {
+        logger.trace("Initializing the ElectroluxAir handler for {}", getThing().getUID());
+
+        config = getConfigAs(ElectroluxAirConfiguration.class);
+        updateStatus(ThingStatus.UNKNOWN);
+
+        scheduler.execute(() -> {
+            Bridge bridge = getBridge();
+            if (bridge != null) {
+                this.bridgeStatusChanged(bridge.getStatusInfo());
+            }
+            Map<String, String> properties = refreshProperties();
+            updateProperties(properties);
+        });
+    }
+
+    @Override
+    public void bridgeStatusChanged(ThingStatusInfo bridgeStatusInfo) {
+        if (bridgeStatusInfo.getStatus() == ThingStatus.ONLINE) {
+            update();
+        }
+        super.bridgeStatusChanged(bridgeStatusInfo);
+    }
```

Comment:
When the bridge comes online the child Thing's `initialize()` is invoked, in which an update can be triggered. Then, this method could be removed.